src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,78 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.MainPage;
+import pages.RegistrationPage;
+import properties.Property;
+import util.RandomSymbolUtil;
+
+
+public class LoginTest extends BaseTest {
+
+    @Test(description = "Positive test to check if the entrance form opens")
+    public void entranceFormTest() {
+        MainPage newMainPage = new MainPage()
+                .openPage()
+                .openEntranceForm();
+        Assert.assertEquals(newMainPage.getLoginViaNickNameMessage(), "или через ник, e-mail", "Message under login field wasn't found.");
```

Comment:
getLoginViaNickNameMessage - make this method more generic so that you can reuse it for other tests. getLoginViaNickNameMessage can be useful for 1-2 tests, what's the point in a separate method for that?
Create a log in method where you pass credentials from the tests

Reply:
done